bin/migrate-oats-data/applications/submissions/sql/statuses/application_decision_released.sql

```diff
@@ -0,0 +1,10 @@
+UPDATE alcs.application_submission_to_submission_status AS app_statuses
+SET
+    effective_date = app.decision_date
```

Comment:
since effective_date is timezone sensitive and the decision_date is UTC the timezone conversion should happen in sql. For example effective_date = app.decision_date AT TIME ZONE 'US/Pacific'. Please apply this to all other status update scripts including NOI

Reply:
not relevant since we using alcs DB here